amrendrakind/math-magicians
# Math Magicians: Tests

**Math Magicians: Tests**

- Set up React Testing Library.
- Write unit tests for the files operate.js and calculate.js using Jest.
- Create unit tests for all React components.
- Run your full tests suite using Jest. All tests should pass.
